s30576-pj-NatanBrzozowski/zad_procesy_i_strumienie.txt

```diff
@@ -0,0 +1,239 @@
+komenda ls -a | grep N wypisała katalog w ktorym znajduje sie "N"
+komenda ls -a | grep N > lista_plikow_z_literka_n.txt znalazła katalog z "N" 
+i zapisała go do pliku
+komenda ls -a | grep N | tee lista_plikow_z_literka_n_2.txt wyswietliła nam 
+znaleziony katalog i zapisała go do pliku
+> przekierowuje dane tylko do pliku a tee umozliwia jednoczesne przekazywanie 
+danych na standardowe wyjscie i do pliku
+
+
+|-----------------------------|
+|  FRAGMENT KODU Z TERMINALA: |
+|-----------------------------|
+natan@DESKTOP-OP762KV MINGW64 ~/repo/s30576-NatanBrzozowski (master)
+$ ls -l
+total 12
+-rw-r--r-- 1 natan 197121 3264 Oct  4 22:22 README.md
+drwxr-xr-x 1 natan 197121    0 Oct 24 23:03 s30576-pj-NatanBrzozowski/
+drwxr-xr-x 1 natan 197121    0 Oct  4 22:24 t0meck-pj-TomaszIdzikowski/
+
+natan@DESKTOP-OP762KV MINGW64 ~/repo/s30576-NatanBrzozowski (master)
+$ ls -l | grep N
+drwxr-xr-x 1 natan 197121    0 Oct 24 23:03 s30576-pj-NatanBrzozowski/
+
+natan@DESKTOP-OP762KV MINGW64 ~/repo/s30576-NatanBrzozowski (master)
+$ ls -a | grep N > lista_plikow_z_literka_n.txt
+
+natan@DESKTOP-OP762KV MINGW64 ~/repo/s30576-NatanBrzozowski (master)
+$ ls -l
+total 13
+-rw-r--r-- 1 natan 197121 3264 Oct  4 22:22 README.md
+-rw-r--r-- 1 natan 197121   27 Nov  6 22:48 lista_plikow_z_literka_n.txt
+drwxr-xr-x 1 natan 197121    0 Oct 24 23:03 s30576-pj-NatanBrzozowski/
+drwxr-xr-x 1 natan 197121    0 Oct  4 22:24 t0meck-pj-TomaszIdzikowski/
+
+natan@DESKTOP-OP762KV MINGW64 ~/repo/s30576-NatanBrzozowski (master)
+$ cat lista_plikow_z_literka_n.txt
+s30576-pj-NatanBrzozowski/
+
+natan@DESKTOP-OP762KV MINGW64 ~/repo/s30576-NatanBrzozowski (master)
+$ ls -a | grep N | tee lista_plikow_z_literka_n_2.txt
+s30576-pj-NatanBrzozowski/
+
+natan@DESKTOP-OP762KV MINGW64 ~/repo/s30576-NatanBrzozowski (master)
+$ ls -l
+total 14
+-rw-r--r-- 1 natan 197121 3264 Oct  4 22:22 README.md
+-rw-r--r-- 1 natan 197121   27 Nov  6 22:48 lista_plikow_z_literka_n.txt
+-rw-r--r-- 1 natan 197121   27 Nov  6 22:49 lista_plikow_z_literka_n_2.txt
+drwxr-xr-x 1 natan 197121    0 Oct 24 23:03 s30576-pj-NatanBrzozowski/
+drwxr-xr-x 1 natan 197121    0 Oct  4 22:24 t0meck-pj-TomaszIdzikowski/
+
+natan@DESKTOP-OP762KV MINGW64 ~/repo/s30576-NatanBrzozowski (master)
+$ cat lista_plikow_z_literka_n_2.txt
+s30576-pj-NatanBrzozowski/
+
+natan@DESKTOP-OP762KV MINGW64 ~/repo/s30576-NatanBrzozowski (master)
+$
+
+--------------------------------------------------------------------------------
+
+|=============================|
+|         ZADANIE 1:          |
+|=============================|
+znak zapytania oznacza ze system nie jest w stanie określić wlasciciela procesu 
+dla danego wpisu
+wszystkie procesy bash wyswietle za pomoca: ps aux | grep bash
+wszystkie procesy należące do użytkownika root wyswietle za pomoca: ps -u root
+
+
+|-----------------------------|
+|  FRAGMENT KODU Z TERMINALA: |
+|-----------------------------|
+natan@DESKTOP-OP762KV MINGW64 ~/repo/s30576-NatanBrzozowski (master)
```

Comment:
To zadanie jest do poprawy.
Wykonujesz je na MINGW, które jest inne od faktycznego linuxa.